src/web/routes/application/confirm.js

```diff
@@ -1,15 +1,26 @@
 const { handleRequestForPath } = require('./middleware')
 const { CONFIRM_URL } = require('./common/constants')
 
-const pageContent = ({ translate }) => ({
-  title: translate('confirm.title'),
-  subTitle: translate('confirm.subTitle')
+const confirmPageContent = req => ({
```

Comment:
Suggestion (non PR blocking): I feel its a bit unnecessary abstracting the page content for each of these render methods into separate functions. They are only used once and return small objects that would be easier to mentally parse if they were inline in the render function.

Reply:
Done.